demo/pages/index.js

```diff
@@ -1,12 +1,47 @@
 import React from 'react';
-import NewComponent from '@moxy/react-lib-template';
+import { Accordion, AccordionItem } from '@moxy/react-accordion';
 
 import styles from './index.module.css';
 
+const Title = () => (
+    <div className={ styles.title }>
+        Title 0
+    </div>
+);
+
+// eslint-disable-next-line react/prop-types
+const Content = ({ isActive }) => (
+    <div className={ styles.description }>
+        { isActive ? 'Open' : 'Closed' }
+    </div>
+);
+
 const Home = () => (
     <div>
```

Comment:
Give this a class and wrap the classNames you're overriding in the accordion, so you don't hit [this issue](https://github.com/moxystudio/react-lib-template#i-cant-override-the-components-css-whats-happening).

```suggestion
    <div className={ styles.home }>
```